plugins/usd/module/vtkF3DFaceVaryingPointDispatcher.cxx

```diff
@@ -0,0 +1,148 @@
+#include "vtkF3DFaceVaryingPointDispatcher.h"
+
+#include "vtkFloatArray.h"
+#include "vtkInformation.h"
+#include "vtkInformationIntegerKey.h"
+#include "vtkInformationVector.h"
+#include "vtkPointData.h"
+
+vtkStandardNewMacro(vtkF3DFaceVaryingPointDispatcher);
+
+vtkInformationKeyMacro(vtkF3DFaceVaryingPointDispatcher, INTERPOLATION_TYPE, Integer);
```

Comment:
F3D_INTERPOLATION_TYPE would be safer then.

Reply:
Why? We already have the class scope: `vtkF3DFaceVaryingPointDispatcher::INTERPOLATION_TYPE()`

Reply:
ha right! I somehow thought the symbol was global.